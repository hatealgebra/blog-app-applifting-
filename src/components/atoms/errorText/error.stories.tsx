import React from 'react';
import ErrorText from './error.styled';

export const ErrorTextExample = () => (
  <ErrorText>This is error text sample</ErrorText>
);

export default {
  title: 'Atoms/Error text',
  component: ErrorText,
};
