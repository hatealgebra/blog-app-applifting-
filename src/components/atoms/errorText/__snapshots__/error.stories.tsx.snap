// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/Error text ErrorTextExample smoke-test 1`] = `
<span class="sc-gKHVLF gugXSe">
  This is error text sample
</span>
`;
